roles/composer/tasks/main.yml

```diff
@@ -9,3 +9,9 @@
   command: composer self-update
   register: composer_update_result
   changed_when: composer_update_result.rc == 0 and 'already using composer version' not in composer_update_result.stderr
+
+- name: add Composer vendor binary path
+  lineinfile:
+    dest: /etc/environment
+    regexp: "^PATH=.*"
+    line: PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:./vendor/bin"
```

Comment:
I don't like hardcoding all these values. Just add a line that is like `PATH="$PATH:./vendor/bin"`